src/applications/personalization/profile/components/connected-apps/AppDeletedAlert.jsx

```diff
@@ -5,13 +5,37 @@ import { focusElement } from 'platform/utilities/ui';
 
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 
-export const AppDeletedAlert = props => {
+export const AppDeletedAlert = ({
+  dismissAlert,
+  id,
+  privacyPolicies,
+  title,
+}) => {
   useEffect(() => {
     focusElement('[data-focus-target]');
   }, []);
-
-  const { id, title, dismissAlert } = props;
-  const alertMessage = `We’ve disconnected ${title}. This app can’t access any new information from your VA.gov profile. Some apps may still store information you’ve already shared. If you’d like to ask the app to delete any stored information, contact the app’s support.`;
+  const privacyLink = (
+    <a
+      href={`${privacyPolicies[title]}`}
+      target="_blank"
+      rel="noopener noreferrer"
```

Comment:
Might be nice to add an `aria-label` that adds the name of the app. But that might be overkill. Perhaps @joshkimux could weigh in on that.

Reply:
Thanks for catching that! I agree that it should be included to meet the 508 standards. Will include in update.